.github/workflows/unit-tests.yml

```diff
@@ -27,19 +28,27 @@ jobs:
           echo "IS_RELEASE=true" >> $GITHUB_OUTPUT
           echo "IS_RELEASE=true" >> $env:GITHUB_OUTPUT
         if: "${{ startsWith(github.head_ref, 'release-') }}"
+
       - name: Git checkout
         uses: actions/checkout@v4
         if: '${{!steps.release-check.outputs.IS_RELEASE}}'
+
       - name: Use Node.js ${{ matrix.node-version }}
         uses: actions/setup-node@v4
         with:
           node-version: ${{ matrix.node-version }}
           cache: npm
           check-latest: true
         if: '${{!steps.release-check.outputs.IS_RELEASE}}'
+
       - name: Install core dependencies
         run: npm ci --no-audit
         if: '${{!steps.release-check.outputs.IS_RELEASE}}'
+
+      - name: Build project
+        run: npm run build
+        if: '${{!steps.release-check.outputs.IS_RELEASE}}'
```

Comment:
IIRC we didn't need build for unit tests? did you check?

Reply:
I copied this out of your changes

https://github.com/netlify/cli/pull/7069/files#diff-cd0effce06f425599e96952f4d5f684641a683f46d9909e7c74a9f8b2f03ccd3R33-R34